src/css/Common.css

```
@@ -2,14 +2,16 @@
   height: 100dvh;
   width: 100vw;
   overflow-y: auto;
+  background-color: transparent;
   background-color: rgb(227, 227, 227);
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

background-color is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)